src/templates/static/_layout/layout.jsx

```diff
@@ -73,6 +74,14 @@ const Layout = () => {
                     <InterviewPopup /> {/* TODO: remove when UX research is finished */}
                     <DerivIFrame />
                     {/* <Elevio /> */}
+                    <div id='language-menu-modal' className='invisible'>
+                        <div id='language-menu'>
+                            <div id='language-menu-header'>
+                                <h3>{it.L('Language Settings')}</h3>
```

Comment:
Same here, please use sentence case.

```suggestion
                                <h3>{it.L('Language settings')}</h3>
```